virtcontainers/api.go

```diff
@@ -20,7 +21,6 @@ import (
 	"github.com/kata-containers/runtime/virtcontainers/store"
 	"github.com/kata-containers/runtime/virtcontainers/types"
 	specs "github.com/opencontainers/runtime-spec/specs-go"
-	opentracing "github.com/opentracing/opentracing-go"
```

Comment:
As we look @ potentially migrating to OpenTelemetry in the future, I appreciate being able to abstract this out of the rest of the code base...